CADETProcess/comparison/comparator.py

```diff
@@ -267,15 +272,16 @@ def setup_comparison_figure(self, plot_individual=False):
 
     def plot_comparison(
         self,
-        simulation_results,
-        axs=None,
-        figs=None,
-        file_name=None,
-        show=True,
-        plot_individual=False,
-        use_minutes=True,
-    ):
-        """Plot the comparison of the simulation results with the reference data.
+        simulation_results: list[SimulationResults],
```

Comment:
I know, I wasn't always super consistent myself, but please use the following indentation style:
```
def foo(
        arg_1: bool False,
        ) -> NoReturn:
    pass
```

Please also check the other methods.